.github/workflows/auto-all-publish.yml

```
@@ -36,25 +36,25 @@ jobs:
           node-version: 20.10.0
           registry-url: 'https://registry.npmjs.org'
-      
+
       - name: Get pnpm store directory
         id: pnpm-cache
         run: |
-            echo "pnpm_cache_dir=$(pnpm store path)" >> $GITHUB_OUTPUT
-  
+          echo "pnpm_cache_dir=$(pnpm store path)" >> $GITHUB_OUTPUT
+
       - uses: actions/cache@v3
```

actionlint (1.7.4)

the runner of "actions/cache@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)